libs/base-ui/components/Layout/Nav/Banner.tsx

```diff
@@ -33,7 +33,7 @@ export default function Banner({ href, text, bannerName }: BannerProps) {
       },
       AnalyticsEventImportance.high,
     );
-  }, [logEvent, ActionType, ComponentType, AnalyticsEventImportance]);
```

Comment:
it's fine to add bannerName, but a) our linter requires exhaustive dependencies, and b) these values are static and don't cause performance impact, so we can leave the rest of them.